Makefile

```diff
@@ -54,6 +54,9 @@ check:                          ## Run required checkers and linters.
 	ansible-playbook --check ansible/playbook/tasks/update.yml
 	ansible-lint ansible/playbook/tasks/update.yml
 
+check-all: check                ## Run golang ci linter to check new changes from master.
```

Comment:
Not from master. We might also adopt a comment from https://github.com/percona/pmm-managed/blob/18868f558c45e3343a5317e75858df51a7da2b49/Makefile.devcontainer#L58-L59